src/module-config.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+module.exports = (arg) => {
+  if (typeof arg === 'function') {
+    return arg
+  } else if (typeof arg === 'object') {
+    const requestAPI = require('./request-api')
+    return requestAPI(arg)
+  } else {
+    throw new Error('Argument must be a send function or a config object.')
+  }
+}
```

Comment:
This doesn't seem to be correct. What if a user does `const node = ipfsApi('/ip4/127.0.0.1/tcp/5001')` ?